src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -111,4 +112,13 @@ class Wallet(
   fun removeAccountSigner(sourceAddress: String, signerAddress: String): Transaction {
     return addAccountSigner(sourceAddress, signerAddress, 0)
   }
+
+  // Submit transaction
+  fun submitTransaction(signedTransaction: Transaction): SubmitTransactionResponse {
```

Comment:
I'm not sure how valuable this method is. The only thing it does is provides a `server` internally.

There are no tests for this because I couldn't get `server` mock working through `wallet`. It worked when I used mocked `server` directly in the test. We probably don't need a test for this, anyway. :woman_shrugging:

Reply:
responded in another comment.